integration_test/third_party_apps_data/applications/dcgm/debian_ubuntu/install

```diff
@@ -43,5 +43,6 @@ sudo apt-get update
 sudo apt-get install -y datacenter-gpu-manager
 sudo systemctl --now enable nvidia-dcgm
 
-# check DCGM service running and load profiling module
-dcgmi discovery --list
+# check if DCGM service is running
+# This is for debugging the test only - ignore return code
```

Comment:
It's not clear from this comment that this is only here for the output. How about the following:
```suggestion
# This command is only used for informational/debugging output.
```